lms/static/js/spec/search/search_spec.js

```diff
@@ -353,6 +365,7 @@ define([
             expect(this.listView.$el).toContainHtml('Search Results');
             expect(this.listView.$el).toContainHtml('this is a short excerpt');
 
+            searchResults[1] = searchResults[0]
```

Comment:
This adds second result to mocked response. It didn't have to be done before because new result set was just appended to list view (this wasn't the best way to handle situation since same results could be added twice by mistake). Now new results are replacing old set here, and therefore to simulate more results I added same entry twice. 
Change was needed as part of this PR because collection had to treat models differently and not reset each time new results were returned (only list is reset), so it could handle link follow event and log it.